### ISSUE TITLE
Idea: Custom Fields in Scheduler Admin Panel

### ISSUE THREAD

**Issue by daniel-samson** [salesagility/SuiteCRM#888]
Hi,

I had an idea. It would be great if there was an easy way to add custom fields to a custom scheduler. So that a user could pass in  options into a scheduled task. Thus determining the behaviour of a custom scheduler.

Just a thought.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Idea: Custom Fields in Scheduler Admin Panel](https://trello.com/c/OVajn11R/119-idea-custom-fields-in-scheduler-admin-panel)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
